src/overlay/ConnectionTests.cpp

```diff
@@ -0,0 +1,73 @@
+// Copyright 2018 Stellar Development Foundation and contributors. Licensed
```

Comment:
new file -> please update Visual C++ project

Reply:
now that I think of it, I think we should just put this new test in the `OverlayTests.cpp` file as it's just another variation of the existing tests (and tests there are basically testing connections).